tools/trap/ramdumpParser.py

```diff
@@ -857,9 +879,13 @@ def main():
 			global g_etext_ram
 			g_etext_ram = rParser.get_address_of_symbol("_etext_ram")
 
-		if ((dump_file != None) and (log_file != None)):
-			# Find the point of crash in the kernel, application or common binaries
-			find_crash_point(log_file, elf)
+		if 'CONFIG_BOARD_CRASHDUMP=y' in data:
+			if ((dump_file != None) and (log_file != None)):
+				# Find the point of crash in the kernel, application or common binaries
+				find_crash_point(log_file, elf)
+		else:
+				find_crash_point(log_file, elf)
```

Comment:
Is this `if` and `else` needed?
Because they have same params.

Reply:
The if part is to check the presence of both log file & ramdump file in case of CONFIG_BOARD_CRASHDUMP=y.
Else is for normal crash point assertion.